release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -1431,3 +1436,13 @@ It is recommended to delete the default StorageClass and allow the
 ClusterStorageOperator to recreate this with the correct attributes after
 the upgrade is completed.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1751641[*BZ#1751641*])
+
+* Non-serverless workloads show resources for serverless workloads in the Topology resources panel. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760810[*BZ#1760810*])
+
+* Pod status is depicted inconsistently in the Topology view, resources side panel, and the deployment configs details page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760827[*BZ#1760827*])
+
+* Deploy image ignores the target port selection in the create application workflow and always uses the first entry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760836[*BZ#1760836*])
+
+* Certain features like the name of the application and the build status are not rendered in the Topology view on the Edge browser. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760858[*BZ#1760858*])
+
+* Determination of active pods when a rollout fails can be incorrect in the Topology view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
```

Comment:
I think we have **Topology** view elsewhere, so we should keep consistent if we're going to embolden it.